CHANGELOG.md

```diff
@@ -20,8 +20,13 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 - Public: Unbundled dependencies for npm
 
 ### Fixed
+<<<<<<< HEAD
```

Comment:
You have unresolved conflict commited here.

Reply:
Eeks! Thanks for spotting this! For some reason this file didn't show up in the merge conflict files on Fork. 🙈